### PR TITLE
Added the AI-on-the-edge-device

### PR DESCRIPTION
**Pull request by caco3**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/energy/electricity-grid.markdown [hunk 1]
```diff
@@ -45,6 +45,12 @@ We have developed [Home Assistant Glow](https://github.com/klaasnicolaas/home-as
 
 ![Photo of Home Assistant Glow attached to an electricity meter](/images/docs/energy/home-assistant-glow.jpg)
 
+#### Read the meter using an AI-on-the-edge-device
+
+[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant discovery functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways. 
+
+<img src='/images/docs/energy/ai-on-the-edge-device.jpg' alt='AI-on-the-edge-device Workflow'>
```
Comment (frenck) [inline source/_docs/energy/electricity-grid.markdown]:
```suggestion
![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/ai-on-the-edge-device.jpg)
```

File: source/_docs/energy/gas.markdown [hunk 2]
```diff
@@ -22,3 +22,9 @@ The P1 port is a standardized port on electricity meters in the Netherlands, Bel
 We have worked with creator [Marcel Zuidwijk](https://www.zuidwijk.com) to develop [SlimmeLezer+](https://www.zuidwijk.com/product/slimmelezer-plus/). It's an affordable P1 reader powered by [ESPHome](https://esphome.io) that will seamlessly integrate this information in Home Assistant. It is being sold on [his website](https://www.zuidwijk.com/product/slimmelezer-plus/) and the firmware is open source [on GitHub](https://github.com/zuidwijk/dsmr).
 
 ![Photo of SlimmeLezer attached to a smart electricity meter](/images/docs/energy/slimmelezer.jpg)
+
+#### Read the Gas Meter using an AI-on-the-edge-device
```
Comment (frenck) [inline source/_docs/energy/gas.markdown:26]:
```suggestion
#### Read the Gas Meter using an AI-on-the-edge-device

```

File: source/_docs/energy/gas.markdown [hunk 3]
```diff
@@ -22,3 +22,9 @@ The P1 port is a standardized port on electricity meters in the Netherlands, Bel
...
+[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways.
```
Comment (frenck) [inline source/_docs/energy/gas.markdown]:
```suggestion
[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways. 
```

File: source/_docs/energy/gas.markdown [hunk 4]
```diff
@@ -22,3 +22,9 @@ The P1 port is a standardized port on electricity meters in the Netherlands, Bel
 We have worked with creator [Marcel Zuidwijk](https://www.zuidwijk.com) to develop [SlimmeLezer+](https://www.zuidwijk.com/product/slimmelezer-plus/). It's an affordable P1 reader powered by [ESPHome](https://esphome.io) that will seamlessly integrate this information in Home Assistant. It is being sold on [his website](https://www.zuidwijk.com/product/slimmelezer-plus/) and the firmware is open source [on GitHub](https://github.com/zuidwijk/dsmr).
 
 ![Photo of SlimmeLezer attached to a smart electricity meter](/images/docs/energy/slimmelezer.jpg)
+
+#### Read the Gas Meter using an AI-on-the-edge-device
+[AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways. 
+
+<img src='/images/docs/energy/ai-on-the-edge-device.jpg' alt='AI-on-the-edge-device Workflow'>
```
Comment (frenck) [inline source/_docs/energy/gas.markdown]:
```suggestion
![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/ai-on-the-edge-device.jpg)
```

File: source/_docs/energy/water.markdown [hunk 5]
```diff
@@ -24,6 +24,11 @@ Alternatively, the following shops sell ESPHome-based devices, that use a proxim
...
 
-Maybe you like to build one yourself? Pieter Brinkman has quite a [nice blog article on how to create your own water sensor](https://www.pieterbrinkman.com/2022/02/02/build-a-cheap-water-usage-sensor-using-esphome-home-assistant-and-a-proximity-sensor/) using ESPHome, or [build a water meter](https://www.ztatz.nl/p1-monitor-watermeter/) that works with the [P1 Monitor](/integrations/p1_monitor) integration.
+Maybe you like to build one yourself?
+ - Pieter Brinkman has quite a [nice blog article on how to create your own water sensor](https://www.pieterbrinkman.com/2022/02/02/build-a-cheap-water-usage-sensor-using-esphome-home-assistant-and-a-proximity-sensor/) using ESPHome, or [build a water meter](https://www.ztatz.nl/p1-monitor-watermeter/) that works with the [P1 Monitor](/integrations/p1_monitor) integration.
+ - [AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways.
```
Comment (frenck) [inline source/_docs/energy/water.markdown]:
```suggestion
 - [AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data in various ways. 
```

File: source/_docs/energy/water.markdown [hunk 6]
```diff
@@ -24,6 +24,11 @@ Alternatively, the following shops sell ESPHome-based devices, that use a proxim
...
+Maybe you like to build one yourself?
+ - Pieter Brinkman has quite a [nice blog article on how to create your own water sensor](https://www.pieterbrinkman.com/2022/02/02/build-a-cheap-water-usage-sensor-using-esphome-home-assistant-and-a-proximity-sensor/) using ESPHome, or [build a water meter](https://www.ztatz.nl/p1-monitor-watermeter/) that works with the [P1 Monitor](/integrations/p1_monitor) integration.
+ - [AI-on-the-edge-device](https://github.com/jomjol/AI-on-the-edge-device) is a project running on an ESP32-CAM and can be fully integrated into Home Assistant using the Home Assistant Discovery Functionality of MQTT. It digitalizes your gas/water/electricity meter display and provides its data through various ways. 
+ 
+<img src='/images/docs/energy/ai-on-the-edge-device.jpg' alt='AI-on-the-edge-device Workflow'>
```
Comment (frenck) [inline source/_docs/energy/water.markdown]:
```suggestion
![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/ai-on-the-edge-device.jpg)
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @caco3 👍

**Comment by caco3** [comment]
@frenck Thank you!

**Pull request merged**